packages/ckeditor5-link/src/utils.js

```diff
@@ -135,3 +138,7 @@ export function isImageAllowed( element, schema ) {
 
 	return element.is( 'element', 'image' ) && schema.checkAttribute( 'image', 'linkHref' );
 }
+
+export function isEmail( linkHref ) {
+	return EMAIL_REG_EXP.test( linkHref );
+}
```

Comment:
A bit PITA, but you can get used to it :)

Every "public" function or class (so here any exported one) needs at least a brief description. Check this file or other similar for inspiration. In most cases, descriptions by PK or Olek would be a good example of such description.

Anyway, you don't have to strees too much on this either as we always double check this on review :smiley: